src/app/options/options.component.ts

```diff
@@ -100,7 +100,37 @@ export class OptionsComponent implements OnInit {
     const modalRef = this.modalService.open(ModalComponent);
     modalRef.componentInstance.title = 'About this project';
     modalRef.componentInstance.content = `
-        <strong>Hi</strong> there!
+        <p>
+            Thank you for giving the official Kubernetes Release Notes website
+            a try! The project aims to provide you with the best possible user
+            experience when it comes to finding out what changed in Kubernetes.
+        </p>
+        <p>
+            We would love to get your feedback about this website, especially
+            in terms of usability, look and feel and how to improve it even
+            further regarding your personal needs. In order to achive this goal
+            we prepared a survey and we would appreciate if you could take your
+            time to fill it out and help us to improve.
+        </p>
+        <p>
+            <a href="#" taget="_bank">
+                👉 Help us and participate in the survey.
+            </a>
+        </p>
+            We target to evaluate the results of the survey at the end of the
+            v1.16 release cycle on the official Kubernetes
+            <a href="https://groups.google.com/forum/#!forum/kubernetes-sig-release"
+               target="_bank">SIG release mailling list</a>.
+        </p>
+        <p>
+            Do you want to contribute or give further feedback? Great, feel
+            free to open up a pull request or issue within the
```

Comment:
. . .open up a pull request or issue in the. . .

Reply:
Fixed :)